cmd/river/rivercli/command.go

```diff
@@ -17,6 +16,16 @@ import (
 	"github.com/riverqueue/river/rivermigrate"
 )
 
+const (
+	// Referencing the offical PostgreSQL documentation, it appears that
+	// `postgresql://` is the default/preferred URI scheme, with `postgres://`
+	// as the fallback. River accepts both.
+	//
+	// See https://www.postgresql.org/docs/current/libpq-connect.html#LIBPQ-CONNSTRING-URIS.
```

Comment:
Thank you for the comment (usually I think these are quite a good idea), but maybe drop this one. It feels oddly unsure of itself ("it appears that"), and also the naming of the constants make it so obvious what's going on here that I don't think it's really needed.